flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptivebatch/DefaultVertexParallelismAndInputInfosDecider.java

```diff
@@ -559,7 +559,7 @@ static DefaultVertexParallelismAndInputInfosDecider from(
                 configuration.get(
                         BatchExecutionOptions.ADAPTIVE_AUTO_PARALLELISM_AVG_DATA_VOLUME_PER_TASK),
                 configuration.get(
-                        BatchExecutionOptions
-                                .ADAPTIVE_AUTO_PARALLELISM_DEFAULT_SOURCE_PARALLELISM));
+                        BatchExecutionOptions.ADAPTIVE_AUTO_PARALLELISM_DEFAULT_SOURCE_PARALLELISM,
```

Comment:
We should also update the description of `ADAPTIVE_AUTO_PARALLELISM_DEFAULT_SOURCE_PARALLELISM`.

Reply:
It's better to change the default value to `noDefaultValue()` to avoid misleading.